app/src/main/java/org/simple/clinic/newentry/PatientEntryScreen.kt

```diff
@@ -432,17 +432,17 @@ class PatientEntryScreen(context: Context, attrs: AttributeSet) : RelativeLayout
     }
   }
 
-  override fun showLengthTooShortPhoneNumberError(show: Boolean) {
+  override fun showLengthTooShortPhoneNumberError(show: Boolean, requiredNumberLength: Int) {
     if (show) {
-      phoneNumberInputLayout.error = context.getString(R.string.patiententry_error_phonenumber_length_less)
+      phoneNumberInputLayout.error = String.format(context.resources.getString(R.string.patiententry_error_phonenumber_length_less), requiredNumberLength)
```

Comment:
`String.format` is the wrong method to use here since it's a JVM method that is unaware of the Android resources framework. We should be using the version of [`Context#getString`](https://developer.android.com/reference/android/content/Context#getString(int,%20java.lang.Object...)) that accepts vararg format arguments instead.